libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1458,6 +1461,21 @@ void AP_OSD_Screen::draw_horizon(uint8_t x, uint8_t y)
     float roll = ahrs.roll;
     float pitch = -ahrs.pitch;
 
+    //correct horizon for TRIM_PITCH_CD if Plane and in fixed wing mode
+#if APM_BUILD_TYPE(APM_BUILD_ArduPlane)
+    float value = 0;
+    bool in_vtol_mode = FALSE;
+    AP_Param* param=AP_Param::get_singleton();
+    QuadPlane* quadplane=QuadPlane::get_singleton();
+    if (quadplane != nullptr) {
+       in_vtol_mode = quadplane->in_vtol_mode();
+    }
+    if (param != nullptr && !in_vtol_mode) {
+        param->get("TRIM_PITCH_CD",value);
```

Comment:
Its probably better to get it once at boot, look up by string is expensive. You can grab a pointer to the param so it will still update in real time.